### PR TITLE
Added Subgraph docs and added to sidebars

### PR DESCRIPTION
**Pull request by PaulieB14**

### CODE REVIEW

File: sidebars/sidebarsDevelopers.js [hunk 1]
```diff
@@ -25,3 +25,12 @@ module.exports = {
     'v2/fork-yearn-ui',
   ],
 }
```
Comment (saltyfacu) [inline sidebars/sidebarsDevelopers.js:27]: You are missing a comma here

File: sidebars/sidebarsDevelopers.js [hunk 2]
```diff
@@ -25,3 +25,12 @@ module.exports = {
...
+{
+  type: 'category',
+  label: 'Subgraph Information',
+  items: [
+    'docs/developers/v2/subgraph info.md',
```
Comment (saltyfacu) [inline sidebars/sidebarsDevelopers.js:32]: Don't use spaces in filenames, use `-`

**Review by saltyfacu — COMMENTED** [review]
Hey @PaulieB14 ! Thanks for this contribution :D I've left 2 comments so the deployment doesn't fail. This is great 🚀

File: sidebars/sidebarsDevelopers.js (hunk 2 above)
Comment (PaulieB14) [reply sidebars/sidebarsDevelopers.js]: Thank you @saltyfacu.  I am not the greatest w/ Sidebars.  I appreciate your feedback.  Should be fixed I believe.  Let me know.

**Comment by MarcoWorms** [comment]
would be cool to link with this pages too! https://github.com/yearn/yearn-devdocs/pull/262/files

**Comment by PaulieB14** [comment]
@MarcoWorms I moved the file you suggested down to my additional subgraph docs.  Let me know if that is what you meant.  Thanks

**Comment by DarkGhost7** [comment]
@PaulieB14 made a pr here https://github.com/PaulieB14/yearn-devdocs/pull/1 to fix some issues but it still is showing something wrong the entities.md tables

**Comment by DarkGhost7** [comment]
If you merge my pr you will still have this issue 
image

**Comment by PaulieB14** [comment]
@DarkGhost7 I merged your fixes.  Let me know if I need anything further

**Comment by PaulieB14** [comment]
@DarkGhost7 I believe I found error in entities.  Let me know if it works.  Thanks.

**Comment by saltyfacu** [comment]
Thanks @PaulieB14. I asked @rareweasel to review it. He's the subgraph expert

File: sidebars/sidebarsDevelopers.js (hunk 2 above)
Comment (saltyfacu) [reply sidebars/sidebarsDevelopers.js]: this is looking good. Is there any other place we can put yDaemon in? I think it has nothing to do with the subgraph

File: docs/developers/v2/subgraph-info.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+## Ethereum
+
+- [Explorer Page](https://thegraph.com/explorer/subgraphs/5xMSe3wTNLgFQqsAc5SCVVwT4MiRb5AogJCuSN9PjzXF?view=Overview)
+- Graphql Endpoint: https://gateway.thegraph.com/api/{api-key}/subgraphs/id/5xMSe3wTNLgFQqsAc5SCVVwT4MiRb5AogJCuSN9PjzXF
+- [Code Repo](https://github.com/yearn/yearn-vaults-v2-subgraph)
+
+## Fantom
+- [Explorer Page](https://thegraph.com/hosted-service/subgraph/yearn/yearn-vaults-v2-optimism)
+- Graphql Endpoint: https://thegraph.com/hosted-service/subgraph/yearn/yearn-vaults-v2-optimism
```
Comment (saltyfacu) [inline docs/developers/v2/subgraph-info.md:17]: the title says `Fantom` but the links say `Optimism`

Comment (PaulieB14) [reply docs/developers/v2/subgraph-info.md]: @saltyfacu I took the liberty to update the fantom links.  Hopefully this is okay.  Let me know if I can help w anything.

**Review by saltyfacu — APPROVED** [review]
great job! Thanks :)

**Pull request merged**